source/support.tex

```diff
@@ -530,251 +530,141 @@
+  // \libheader{unordered_map}, \libheader{unordered_set}, \libheader{vector}
+#define @\defnlibxname{cpp_lib_not_fn}@                            201603L // also in \libheader{functional}
+#define @\defnlibxname{cpp_lib_null_iterators}@                    201304L // also in \libheader{iterator}
+#define @\defnlibxname{cpp_lib_optional}@                          201606L // also in \libheader{optional}
+#define @\defnlibxname{cpp_lib_parallel_algorithm}@                201603L // also in \libheader{algorithm}, \libheader{numeric}
+#define @\defnlibxname{cpp_lib_quoted_string_io}@                  201304L // also in \libheader{iomanip}
+#define @\defnlibxname{cpp_lib_ranges}@                            201811L
+  // also in \libheader{algorithm}, \libheader{functional}, \libheader{iterator}, \libheader{memory}, \libheader{ranges}
+#define @\defnlibxname{cpp_lib_raw_memory_algorithms}@             201606L // also in \libheader{memory}
+#define @\defnlibxname{cpp_lib_result_of_sfinae}@                  201210L // also in \libheader{functional}, \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_robust_nonmodifying_seq_ops}@       201304L // also in \libheader{algorithm}
+#define @\defnlibxname{cpp_lib_sample}@                            201603L // also in \libheader{algorithm}
+#define @\defnlibxname{cpp_lib_scoped_lock}@                       201703L // also in \libheader{mutex}
+#define @\defnlibxname{cpp_lib_semaphore}@                         201907L // also in \libheader{semaphore}
+#define @\defnlibxname{cpp_lib_shared_mutex}@                      201505L // also in \libheader{shared_mutex}
+#define @\defnlibxname{cpp_lib_shared_ptr_arrays}@                 201611L // also in \libheader{memory}
+#define @\defnlibxname{cpp_lib_shared_ptr_weak_type}@              201606L // also in \libheader{memory}
+#define @\defnlibxname{cpp_lib_shared_timed_mutex}@                201402L // also in \libheader{shared_mutex}
+#define @\defnlibxname{cpp_lib_source_location}@                   201907L // also in \libheader{source_location}
+#define @\defnlibxname{cpp_lib_spaceship}@                         201907L // also in \libheader{compare}
+#define @\defnlibxname{cpp_lib_string_udls}@                       201304L // also in \libheader{string}
+#define @\defnlibxname{cpp_lib_string_view}@                       201606L // also in \libheader{string}, \libheader{string_view}
+#define @\defnlibxname{cpp_lib_three_way_comparison}@              201711L // also in \libheader{compare}
+#define @\defnlibxname{cpp_lib_to_array}@                          201907L // also in \libheader{array}
+#define @\defnlibxname{cpp_lib_to_chars}@                          201611L // also in \libheader{charconv}
+#define @\defnlibxname{cpp_lib_transformation_trait_aliases}@      201304L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_transparent_operators}@             201510L // also in \libheader{memory} ,\libheader{functional}
```

Comment:
Looks like the comma is the wrong side of the space here.

Reply:
Fixed.